client/src/styles/_bootstrap-variables.scss

```diff
@@ -50,16 +50,15 @@
 //
 
 $white:  #fff !default; // bootstrap default
-$gray-700: #4f5861; // old $gray-dark
-$gray-100: #f8f9fa !default; // bootstrap default
-$gray-200: lighten($gray-700, 57%); // old $gray-lighter
-$gray-300: lighten($gray-700, 50%); // old $gray-light
-$gray-400: #ced4da !default; // bootstrap default
-$gray-500: #adb5bd !default; // bootstrap default
-$gray-600: lighten($gray-700, 20%); // old $gray
-$gray-650: lighten($gray-700, 10%);
-$gray-800: darken($gray-700, 10%); // old $gray-darker
-$gray-900: darken($gray-700, 20%); // old $gray-darkest
+$gray-700: #43536d !default;
+$gray-100: lighten($gray-700, 60%) !default;
+$gray-200: lighten($gray-700, 50%) !default;
+$gray-300: lighten($gray-700, 40%) !default;
+$gray-400: lighten($gray-700, 30%) !default;
+$gray-500: lighten($gray-700, 20%) !default;
+$gray-600: lighten($gray-700, 10%) !default;
+$gray-800: darken($gray-700, 10%) !default;
+$gray-900: darken($gray-700, 20%) !default;
```

Comment:
I don't mind using `lighten()` and `darken()` in variables - could we put the calculated colour codes next to them in comment?
Makes searching for colour codes much much easier e.g. especially when we get design details like `#ced4da` and we have no idea which one that is... :/

Reply:
I could add them. I am setting up an easier way to get all the colours through this tool which syncs with the design doc and should match the variables sheet also https://brand.ai/silverstripe/cms-library

It will all be clearer with the pattern library too—when we have it.

Reply:
@flamerohr Are you saying you should put a color code in a comment where there's color calculation or just this section? I can see it being really helpful for searching and debugging purposes but it'd be a pain to have to do that everywhere.

Reply:
Probably just on these grays it would be okay.

Reply:
just on these greys would already be a big help :)